app/components/candidate_interface/degree_grade_component.rb

```diff
@@ -37,11 +37,15 @@ def hint_helper
     t('application_form.degree.grade.hint.not_completed') unless model.completed?
   end
 
+  def specific_grade_options?
+    model.masters? || model.bachelors?
+  end
```

Comment:
We think UK foundation degrees are usually Pass, Merit or Distinction – so we could add those as radio options (but still with an "other" just in case).  That could be a follow-up PR, or one for the backlog though? cc @MylesJarvis 

Not sure how we (and by that I mean mostly me) missed this when we first implemented it... 🤦

Reply:
It looks like there was code there to enter free text grades but it was being bypassed. Also it didn't save the correct value to the database. Should be fixed now.